br/pkg/task/restore.go

```diff
@@ -1358,6 +1365,24 @@ func Exhaust(ec <-chan error) []error {
 	}
 }
 
+func checkTableExistence(ctx context.Context, mgr *conn.Mgr, tables []*metautil.Table, glue glue.Glue) error {
+	if glue.GetType() != "ExecGlue" {
+		return nil
+	}
+	message := "table already exists: "
+	allUnique := true
+	for _, table := range tables {
+		if _, err := mgr.GetDomain().InfoSchema().TableByName(ctx, table.DB.Name, table.Info.Name); err == nil {
```

Comment:
Also you may check whether the err is `NotExist`?

Reply:
That is, when encountered error other than `ErrTableNotExists`, we should return them instead of check next table.